onnx/onnx_cpp2py_export/defs.pyi

```diff
@@ -73,20 +100,32 @@ class OpSchema:
         NonDifferentiable: "OpSchema.DifferentiationCategory" = ...
 
     class FormalParameter:
+        def __init__(
+            self,
+            name: str,
+            type_str: str,
+            description: str = "",
+            param_option: OpSchema.FormalParameterOption = OpSchema.FormalParameterOption.Single,  # noqa: F821
+            is_homogeneous: bool = True,
+            min_arity: int = 1,
+            differentiation_category: OpSchema.DifferentiationCategory = OpSchema.DifferentiationCategory.Unknown,  # noqa: F821
+        ) -> None: ...
         @property
         def name(self) -> str: ...
         @property
-        def types(self) -> Set[str]: ...
+        def types(self) -> set[str]: ...
         @property
-        def typeStr(self) -> str: ...
+        def type_str(self) -> str: ...
```

Comment:
The name follows python convention but i wonder if we should keep the old name for a couple of releases with a warning saying it is going to be deprecated. Some tools may break if they use these names to automate documentation...

Reply:
I did keep the old names. I just removed them from the pyi file so people will not discover them easily.